tests/validation_tests/test_pre_checks.py

```diff
@@ -256,26 +256,59 @@ def test_full_pre_transformation_validation_pipeline_failure(valid_workbook):
     ]
 
 
-def test_full_pre_transformation_validation_pipeline_failure_place_vs_fund(valid_workbook_round_four):
+@pytest.fixture
+def mocked_project_admin_sheet(valid_workbook_round_four, request):
     valid_workbook_round_four["2 - Project Admin"][4] = [
         "",
         "",
         "",
         "",
         "",
-        "Future_High_Street_Fund",
-        "Newark",
+        request.param["fund"],
+        request.param["place"],
         "",
         "",
     ]
-
+    return valid_workbook_round_four
+
+
+@pytest.mark.parametrize(
+    "mocked_project_admin_sheet, expected",
+    [
+        (
+            {"fund": "Future_High_Street_Fund", "place": "Newark"},
+            [
+                WrongInputFailure(
+                    value_descriptor="Place Name vs Fund Type",
+                    entered_value="HS",
+                    expected_values={"TD"},
+                )
+            ],
```

Comment:
I think it's worth adding a comment to explain that Newark is only valid for TD

Reply:
Added and pushed